src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -305,42 +305,82 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
+                                        *ngIf="(correctionRound == 0 || value.results?.[correctionRound - 1]?.completionDate) &&
+                                        (newManualResultAllowed || (value.results?.[correctionRound]?.assessmentType && value.results[correctionRound].assessmentType !== AssessmentType.AUTOMATIC))"
+                                    >
+                                        <a
+                                            [routerLink]="getAssessmentLink(value, correctionRound)"
+                                            [queryParams]="{ 'correction-round': correctionRound }"
+                                            class="btn btn-sm me-1 mb-2"
+                                            [class.btn-success]="(!value.results?.[correctionRound]?.assessmentType || value.results?.[correctionRound].assessmentType === AssessmentType.AUTOMATIC) && !value.results?.[0]?.hasComplaint"
+                                            [class.btn-primary]="value.results?.[correctionRound]?.completionDate || value.results?.[correctionRound]?.hasComplaint"
+                                            [class.btn-warning]="value.results?.[correctionRound] && !value.results[correctionRound].completionDate && !value.results[correctionRound].hasComplaint"
+                                        >
+                                            <fa-icon [icon]="faFolderOpen" [fixedWidth]="true"></fa-icon>
+                                            <ng-container *ngIf="!value.results?.[correctionRound]?.hasComplaint; else complaint">
+                                                {{ ('artemisApp.assessment.dashboard.actions.examCorrectionRound.' +
+                                                (!value.results?.[correctionRound]?.assessmentType || value.results?.[correctionRound].assessmentType === AssessmentType.AUTOMATIC ? 'assess' : (value.results?.[correctionRound]?.completionDate ? 'open' : 'continue')))
+                                                | artemisTranslate: { correctionRound: correctionRound + 1 }
+                                                }}
+                                            </ng-container>
+                                            <ng-template #complaint>
+                                                {{ 'artemisApp.exerciseAssessmentDashboard.showComplaint' | artemisTranslate }}
+                                            </ng-template>
+                                        </a>
+                                        <button
+                                            *ngIf="
                                         newManualResultAllowed &&
-                                        value.results[0]?.assessmentType &&
+                                        value.results?.[0]?.assessmentType &&
                                         !value.results[0].completionDate &&
                                         value.results[0].assessmentType !== AssessmentType.AUTOMATIC
```

Comment:
```suggestion
                                        value.results?.[correctionRound]?.assessmentType &&
                                        !value.results[correctionRound].completionDate &&
                                        value.results[correctionRound].assessmentType !== AssessmentType.AUTOMATIC
```

Needed? If not, I think it might be hard to understand when what is needed why.